test/EFCore.PG.FunctionalTests/BuiltInDataTypesNpgsqlTest.cs

```diff
@@ -38,12 +41,11 @@ var results
                         .ToList();
 
                 Assert.Equal(0, results.Count);
-                Assert.Equal(
+
+                AssertSql(
                     @"SELECT m.""Int""
 FROM ""MappedNullableDataTypes"" AS m
-WHERE (m.""TimeSpanAsTime"" = TIME '00:01:02') AND (m.""TimeSpanAsTime"" IS NOT NULL)",
-                    Sql,
-                    ignoreLineEndingDifferences: true);
+WHERE (m.""TimeSpanAsTime"" = TIME '00:01:02') AND (m.""TimeSpanAsTime"" IS NOT NULL)");
```

Comment:
EF generates stupid expressions. Everybody knows that `NULL` equals nothing even `NULL`.

Reply:
Yes, there is still some known work on null semantics that didn't make it into 3.0.